### PR TITLE
New landing page for docs site

### PR DESCRIPTION
**Pull request by ivanlei**
# New landing page for docs site
In advance of... later today... there was a request to update the homepage for https://doc.agoric.com 
Santi provided this template image and a list of URLs for links.
Agoric_DocHomePage_v1

# Description of changes
We use [veupress](https://vuepress.vuejs.org/) to generate a static docs site. Most changes involve doing vuepress-y type things to create the new look/feel/interaction.

## HomeButtonHeader Component
Makes one of these Screenshot 2023-09-30 at 10 57 04 AM

## HomeButtonRow Component
Makes on of these with either 3 or 6 buttons.
Screenshot 2023-09-30 at 10 57 38 AM

## Faster Mobile Loads & Improved SEO According to Chrome Lighthouse
Added async load of googletagmanager and a meta tag for viewport. I just do what the Lighthouse tells me...

## CSS
Wowsa CSS is nicer than the last time I put my brain into it. Used [mozilla site](https://developer.mozilla.org/en-US/docs/Web/CSS) to read up and used google inspector a LOT to try and figure out if the CSS I added was doing what I lot.

## Node 17.17.x to Avoid PR Failures
See [this failed run](https://github.com/Agoric/documentation/actions/runs/6363551196/job/17279239990?pr=857) for details. Googling about it looks like the instances I have seen of this error are when node published a usable but slightly not so well packaged up set of binaries. I didn’t dive too deep.
But rather than using 18.x in CI I explicitly asked for 18.17.x. 18.18.0 has been out for about 12 days or something and it might be that it was published wonky again. 

# Testing
Tested in Chrome and Brave. In Chrome used all device/screen size permutations available. So it looks good on iOS and Android & even surface.

# Where I Fell Short
- Really couldn't get the sidebar to appear on the homepage. That's a quirk of vuepress and after about an hour trying split between 2 days I didn't succeed. Best I got was a big empty space a sidebar could have been.

### CODE REVIEW

File: main/README.md [hunk 1]
```diff
@@ -4,53 +4,109 @@
...
-    details: Agoric empowers individuals to securely execute transactions, establish new markets, and craft novel patterns of exchange — without centralized control.
-
-  - title: Better security architecture with OCaps
-    details: Agoric uses an object-capability model (OCaps) security architecture, in which access to a programming object itself is the authority to use the object.
-
-  - title: Securely Create and Transfer Assets
-    details: All kinds of digital assets can be easily created, but importantly, they can be transferred in exactly the same ways, with exactly the same security properties.
-
-footer: Apache-2.0 Licensed | Copyright © 2022 - Agoric
+home: false # use default home page layout (hero image with text, features section)
```
Comment (dckc) [inline main/README.md:7]: drop the comment? (or update it)

File: main/README.md [hunk 2]
```diff
@@ -4,53 +4,109 @@
...
+    title2="Getting Started"
+    text2="Set up your environment and start building apps"
+    link2="/guides/getting-started/"
+    title3="Contract Framework"
+    text3="Set up your environment and start building"
+    link3="/guides/zoe/"
   />
+  <HomeButtonRow
+    title1="Token Standards"
+    text1="Understand the ins and outs of Agoric's ERTP standard"
```
Comment (dckc) [inline main/README.md:33]: "ERTP standard" seems a little like ATM machine. ERTP is a protocol. Electronic Rights Transfer Protocol.

```suggestion
    title1="Token Protocol"
    text1="Understand the ins and outs of Agoric's Electronic Rights Transfer Protocol (ERTP)"
```

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "node": "^16.13 || ^18.12"
   },
   "scripts": {
-    "docs:dev": "NODE_OPTIONS=--openssl-legacy-provider vuepress dev main",
+    "docs:dev": "vuepress dev main",
```
Comment (dckc) [inline package.json:10]: Is this intentional?

File: main/README.md [hunk 4]
```diff
@@ -4,53 +4,109 @@
...
+    link1="/guides/ertp/"
+    title2="Samples"
+    text2="Look through our smart contract and dapp examples"
+    link2="/guides/zoe/contracts/"
+    title3="Integrations"
+    text3="Browse through our catalogue of integrations"
+    link3="/guides/chainlink-integration.html#overview"
```
Comment (dckc) [inline main/README.md:40]: chainlink? oh. weird. re-used the filename. odd.

File: main/README.md [hunk 5]
```diff
@@ -4,53 +4,109 @@
...
   />
+  <HomeButtonRow
+    title1="Token Standards"
+    text1="Understand the ins and outs of Agoric's ERTP standard"
+    link1="/guides/ertp/"
+    title2="Samples"
+    text2="Look through our smart contract and dapp examples"
+    link2="/guides/zoe/contracts/"
+    title3="Integrations"
+    text3="Browse through our catalogue of integrations"
```
Comment (dckc) [inline main/README.md:39]: hm. how about:

```suggestion
    text3="Integrate wallets etc. with Agoric like you would with other Cosmos SDK chains."
```

File: main/README.md [hunk 6]
```diff
@@ -4,53 +4,109 @@
...
+    title2="Keplr Wallet"
+    text2="A native wallet running with HardenedJS under the hood"
+    link2="https://www.keplr.app/download"
+    title3="Component Library"
+    text3="Pre-built smart contracts for DeFi, NFTs, and cross-chain!"
+    link3="https://components.agoric.com/"
+  />
+  <HomeButtonRow
+    title1="Cosmos SDK"
+    text1="Our battle-tested consensus mechanism"
```
Comment (dckc) [inline main/README.md:62]: "Our" takes more credit than I'm comfortable with. Suggest:

```suggestion
    text1="The battle-tested consensus mechanism we build on"
```

File: main/README.md [hunk 7]
```diff
@@ -4,53 +4,109 @@
...
+    link1="https://docs.cosmos.network/"
+    title2="IBC"
+    text2="The protocol ensuring Agoric is interoperable with 60÷ chains"
```
Comment (dckc) [inline main/README.md:65]: typo?

```suggestion
    text2="The protocol ensuring Agoric is interoperable with 60+ chains"
```

File: main/README.md [hunk 8]
```diff
@@ -4,53 +4,109 @@
...
+    link2="https://ibc.cosmos.network/main"
+    title3="Bounties"
+    text3="A rotating list of incentivized bounties to grow our platform"
+    link3="https://components.agoric.com/bounties/open-bounties"
```
Comment (dckc) [inline main/README.md:69]: Placing "Bounties" here next to Cosmos and IBC seems odd.

I suggest swapping it with Integrations.

File: main/README.md [hunk 9]
```diff
@@ -4,53 +4,109 @@
...
-Building on 30 years of experience, Agoric is developing a secure distributed object-capabilities (OCap) platform for smart contracts and market-oriented programming. Our platform supports the development of smart contracts and market institutions across many different scales, from large public blockchains to small two-party contracts.
+<div class="home-section">
+  <HomeButtonHeader
+      title="Ready to learn more?"
+      text="Once you've completed the Getting Started, here are a few next steps."
+    />
+  <h3 style="margin-top: 0em;">
+    <a href="/guides/js-programming/hardened-js.html">Learn the basics of HardenedJS</a>
+  </h3>
+  It's just JavaScript but with extra safety features - see why MetaMask uses it to safely run their Snaps technology for 30M+ customers.
```
Comment (dckc) [inline main/README.md:81]: I don't think "extra" is the way we describe it. It's more like: we removed the insecure stuff.

```suggestion
  It's just JavaScript but with hazardous stuff turned off - see why MetaMask uses it to safely run their Snaps technology for 30M+ customers.
```

But I suppose `harden()` _is_ an extra. Hm.

**Review by dckc — COMMENTED** [review]
lots of good stuff here.

I have some suggestions to consider.

File: main/README.md (hunk 1 above)
Comment (ivanlei) [reply main/README.md]: Removed comment

**Review by dckc — APPROVED** [review]
regardless of how my suggestions are dealt with, this is a substantial improvement

File: main/README.md (hunk 8 above)
Comment (ivanlei) [reply main/README.md]: Swapped

File: main/README.md (hunk 7 above)
Comment (ivanlei) [reply main/README.md]: OIC. Thank you!

**Pull request merged**